client/src/css/ClassCard.scss

```diff
@@ -1,18 +1,27 @@
+@import '../styles/colors.scss';
+
 .class-card {
   flex: none;
   width: 175px !important;
+  box-shadow: none !important;
+  border-radius: 15px !important;
+  margin: 0% !important;
 
   .card-content-container {
-    display: flex;
-    justify-content: space-evenly;
+    display: flex !important;
+    justify-content: space-between !important;
   }
 
   .delete-button {
-    height: 25px;
-    width: 25px;
+    height: 25px !important;
+    width: 50px !important;
+    font-family: Tommy !important;
+    padding: 0px;
   }
 
   .class-text {
-    font-size: 200%;
+    transition: all 0.3s ease !important;
```

Comment:
You can also remove the `transition: all 0.3s ease !important;` line because your card isn't using this animation

Reply:
Ahh I see